### PR TITLE
SE-11430

### PR DESCRIPTION
**Pull request by valkyrie69**
private ports default to 8091 and 8092
And other minor edits

### CODE REVIEW

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 1]
```diff
@@ -53,37 +58,38 @@ The following DNS records for your applications are exposed when they are deploy
...
-|*mule-worker-myapp.cloudhub.io* |The external IP address of the Mule workers. HTTP services are exposed on: `${http.port}` and `${https.port}`
-|*mule-worker-internal-myapp.cloudhub.io* |The internal IP address of the Mule workers. The IPs for this DNS record are only accessible within a customer's private xref:virtual-private-cloud.adoc[Anypoint VPC]. *These IPs are not accessible for workers running in the MuleSoft shared VPC.* Public HTTP services are exposed on `${http.port}` and `${https.port}.` Internal HTTP services are exposed on ports *8091/8092*.
+|*mule-worker-myapp.cloudhub.io* |The external IP address of the Mule workers. Public HTTP services are exposed on: `${http.port}` and `${https.port}`
+|*mule-worker-internal-myapp.cloudhub.io* |The internal IP address of the Mule workers. The IP addresses for this DNS record are accessible only within a customer's private xref:virtual-private-cloud.adoc[Anypoint VPC]. *These IPs are not accessible for workers running in the MuleSoft shared VPC.* Internal HTTP services are exposed on: `${http.private.port}` and `${https.private.port}`. The default assigned ports are 8091 and 8092 respectively, and cannot be changed.
```
Comment (nishantkakar) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]:
```suggestion
|*mule-worker-internal-myapp.cloudhub.io* |The internal IP address of the Mule workers. The IP addresses for this DNS record are accessible only within a customer's private xref:virtual-private-cloud.adoc[Anypoint VPC]. *These IP addresses are not accessible for workers running in the MuleSoft shared VPC.* Internal HTTP services are exposed on: `${http.private.port}` and `${https.private.port}`. The default assigned ports are 8091 and 8092 respectively, and cannot be changed.
```

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 2]
```diff
@@ -103,21 +109,21 @@ Depending on what region you deploy your application in, the DNS record and the
 
 |===
 
-Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Europe, the DNS records for internal and external IPs are `mule-worker-myapp.eu.cloudhub.io` and `mule-worker-internal-myapp.eu.cloudhub.io`.
+Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Europe, the DNS records for internal and external IP addresses are `mule-worker-myapp.eu.cloudhub.io` and `mule-worker-internal-myapp.eu.cloudhub.io`.
```
Comment (nishantkakar) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]:
```suggestion
Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Ireland, the DNS records for internal and external IP addresses are `mule-worker-myapp.ir-e1.cloudhub.io` and `mule-worker-internal-myapp.ir-e1.cloudhub.io`.
```

**Review by nishantkakar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/cloudhub-networking-guide.adoc (hunk 1 above)
Comment (george-hoffer) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]: Our translation style is to only put field items in bold and not to use bold for emphasis per "*These IP addresses are not accessible for workers running in the MuleSoft shared VPC.*"  Please use a NOTE instead. Same with the bolds in the first column -- those aren't field names, so no bolding.

**Review by nishantkakar — APPROVED** [review]

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 3]
```diff
@@ -103,21 +109,21 @@ Depending on what region you deploy your application in, the DNS record and the
...
 
 == Firewall Rules and Port Access
 
-The only two ports exposed externally are `${http.port}` and `${https.port}` (by default 8081 and 8082 respectively). If you wish to access other ports, you can do so through the xref:virtual-private-cloud.adoc[*Anypoint Virtual Private Cloud (Anypoint VPC)*] offering.
+The only two ports exposed externally are `${http.port}` and `${https.port}` (by default 8081 and 8082 respectively). If you want to access other ports, you can do so through the xref:virtual-private-cloud.adoc[*Anypoint Virtual Private Cloud (Anypoint VPC)*] offering.
```
Comment (george-hoffer) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]: Another use of bold for emphasis: [*Anypoint Virtual Private Cloud (Anypoint VPC)*] -- remove the stars

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 4]
```diff
@@ -103,21 +109,21 @@ Depending on what region you deploy your application in, the DNS record and the
...
 
-Additional ports can be opened inside the Anypoint VPC, for example, for JMX based monitoring. In order  to do so, refer to the Firewall rules section in out xref:virtual-private-cloud.adoc#firewall-rules[Anypoint VPC documentation].
+Additional ports can be opened inside the Anypoint VPC, for example, for JMX based monitoring. In order  to do so, refer to the Firewall Rules section in xref:virtual-private-cloud.adoc#firewall-rules[Anypoint VPC documentation].
```
Comment (george-hoffer) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]: Too verbose: please change "In order  to do so, refer" to "Refer" or "See"

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 5]
```diff
@@ -35,14 +40,14 @@ Here is an example of a Mule configuration that utilizes this to expose an HTTPS
...
+If other values for `http.port` and `https.port` are specified in the `mule-app.properties` file, these are overwritten at deployment time.
 
 [NOTE]
 --
-For each request that a client makes through CloudHub's load balancer (_myapp.cloudhub.io_), the load balancer maintains two connections. One connection ​is from​ the client and one is to your worker.  For each connection, the load balancer manages an idle timeout of 300 seconds that is triggered when no data is sent over ​either​ connection.  If no data is sent or received during this time period, the load balancer closes ​both connections.
+For each request a client makes through CloudHub's load balancer (_myapp.cloudhub.io_), the load balancer maintains two connections. One connection ​is from​ the client and one is to your worker.  For each connection, the load balancer manages an idle timeout of 300 seconds that is triggered when no data is sent over ​either​ connection. If no data is sent or received during this time period, the load balancer closes ​both connections.
```
Comment (george-hoffer) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]: Italics are not supported in our style. Please cast in tick marks.

File: modules/ROOT/pages/cloudhub-networking-guide.adoc [hunk 6]
```diff
@@ -18,12 +18,17 @@ CloudHub provides a load balancing service for all integrations. You can use the
 
 The CloudHub load balancing service does round robin load distribution across workers, which allows workers to scale linearly as they receive more requests. The load balancing service also provides transparent switchover when an application is upgraded. See xref:managing-applications-on-cloudhub.adoc[Managing Applications on CloudHub].
 
-Each application deployed on CloudHub has a `CNAME` record that refers to the load balancer, for example `myapp.cloudhub.io`. Mule applications deployed on CloudHub must listen on `0.0.0.0` and ports assigned by CloudHub for HTTP and HTTPS. 
+Each application deployed on CloudHub has a `CNAME` record that refers to the load balancer, for example `myapp.cloudhub.io`. Mule applications deployed on CloudHub must listen on `0.0.0.0` and the ports assigned by CloudHub for HTTP and HTTPS.
```
Comment (george-hoffer) [inline modules/ROOT/pages/cloudhub-networking-guide.adoc]: This is incorrect wording in that it sounds like an app can listen on more than one port. Please change wording to "Mule apps deployed in CloudHub must listen on host `0.0.0.0` and to either the HTTP port (8081) or to the HTTPS port (8082)."

**Review by george-hoffer — APPROVED** [review]

**Pull request merged**